src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsService.kt

```diff
@@ -405,6 +418,14 @@ class AdjudicationsService(
     }
   }
 
+  // TODO future change Remove logging and use value only from mapping service once satisfied location picked up correctly
```

Comment:
This is just a temporary change to log the data if there are any failures - once we are confident all is ok, or looking ok...I will remove references to the deprecated field